app/src/main/res/layout/create_playlist_dialog.xml

```diff
@@ -16,7 +15,7 @@
         android:text="@string/new_playlist" />
 
     <EditText
-        android:id="@+id/etPlaylistName"
+        android:id="@+id/etNewPlaylistName"
```

Comment:
the naming convention for xml files is [snake_case](https://en.wikipedia.org/wiki/Snake_case) like on line 15